src/Constants/Constants.php

```diff
@@ -9,5 +9,11 @@
 const COMPATIBILITY_FORWARD_TRANSITIVE = 'FORWARD_TRANSITIVE';
 const COMPATIBILITY_FULL = 'FULL';
 const COMPATIBILITY_FULL_TRANSITIVE = 'FULL_TRANSITIVE';
-
 const VERSION_LATEST = 'latest';
+const ACCEPT_HEADER = ['Accept' => 'application/vnd.schemaregistry.v1+json'];
+const CONTENT_TYPE_HEADER = ['Content-Type' => 'application/vnd.schemaregistry.v1+json'];
+const MEDIA_TYPE_HEADER = [
+    ACCEPT_HEADER,
+   CONTENT_TYPE_HEADER
+];
```

Comment:
This would result in nested arrays - so I guess we don't really need the combined `MEDIA_TYPE_HEADER`. We can just use `ACCEPT_HEADER + CONTENT_TYPE_HEADER` in requests that need both.